pages/windows/ftp.md

```diff
@@ -0,0 +1,35 @@
+# ftp
+
+> Interactively transfer files between a local and remote FTP server.
+
+- Connect to a remote FTP server interactively:
+
+`ftp {{host}}`
+
+- Log in as an anonymous user:
+
+`ftp -A {{host}}`
+
+- Disable automatic login upon initial connection:
+
+`ftp -n {{host}}`
+
+- Run a file containing a list of FTP commands:
+
+`ftp -s:{{path/to/file}} {{host}}`
+
+- Download multiple files (glob expression):
+
+`mget {{*.png}}`
```

Comment:
Is `mget` part oft he same command?

Reply:
`mget`, `mput` and `mdelete` are used once the interactive shell has been started.

Reply:
Ahhh I see.